src/main/java/salesforce/ui/pages/lightning/worktype/NewWorkTypePage.java

```diff
@@ -0,0 +1,76 @@
+package salesforce.ui.pages.lightning.worktype;
+
+import core.selenium.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import salesforce.ui.pages.BasePage;
+
+
```

Comment:
I think should exist one line between imports and class and also is missing javadoc in the class because the WebElementAction class have a javadoc